### PR TITLE
Added email notifications for buyer and broker

### PR DESCRIPTION
**Pull request by rommeltorquator**
#### Implemented the email notification features for broker and buyer accounts

[WIP] Buyer and Broker account creation using the Admin account
[WIP] Buyer and Broker account modification using the Admin account

## The Story
We are now almost finish with the minimum feature requirements for the Stock App. Both buyers and brokers now have email notifications based on the specific requirements. Here is what I did in the PR.
- Added a notification email for newly created broker and buyer accounts
- Added an email notification for newly approved broker
- Modified the requested changes by Miss Analyn

#### Broker email notification for account approval
image
#### Broker account creation email notification
image
#### Buyer account creation email notification
image

## Resources
https://www.youtube.com/watch?v=yqxORosQQbo&t=468s
https://guides.rubyonrails.org/action_mailer_basics.html

### CODE REVIEW

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]
Rebase your branch against the updated master
`git rebase origin/master`

**Comment by rommeltorquator** [comment]
All merge conflicts have been successfully resolved. The remote and local repositories are now in-sync.

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -17,26 +17,39 @@
...
+                                   :broker => broker,
+                                   :approved => true }, method: 'patch' %>
+        <% else %>
+          <h3>Status: Approved</h3>
+        <% end %>
     </div>
-  </div>
-  
 <% end %>
+<% end %>
```
Comment (analyn-cajocson) [inline app/views/home/index.html.erb]: add new line here

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: app/views/home/index.html.erb (hunk 1 above)
Comment (rommeltorquator) [reply app/views/home/index.html.erb]: The change has been made. :)

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**